twython/api.py

```diff
@@ -192,15 +196,16 @@ def _request(self, url, method='GET', params=None, api_call=None):
                 error_message,
                 error_code=response.status_code,
                 retry_after=response.headers.get('X-Rate-Limit-Reset'))
-
+        content=""
         try:
             if response.status_code == 204:
                 content = response.content
             else:
                 content = response.json()
         except ValueError:
-            raise TwythonError('Response was not valid JSON. \
-                               Unable to decode.')
+            if response.content!="":
```

Comment:
Can you make this line:

`if response.content != ''` (spaces around operator and single quotes